apps/docs/content/guides/database/connecting-to-postgres.mdx

```diff
@@ -26,384 +20,121 @@ Supabase provides auto-updating Data APIs. These are the easiest way to get star
 - [GraphQL](/docs/guides/graphql/api): interact with your database through a GraphQL interface.
 - [Realtime](/docs/guides/realtime#realtime-api): listen to database changes over websockets.
 
-## Connection pooler
+## Client libraries
 
-A "connection pool" is a system (external to Postgres) which manages Postgres connections.
+Supabase offers [client libraries](https://supabase.com/docs#client-libraries) for popular programming languages. These libraries provide a convenient wrapper around the Data APIs, making it easier to interact with your database using your preferred language.
 
-When a client makes a request, the pooler "allocates" an available connection to the client. When the client transaction or session is completed the connection is returned to the pool and is free to be used by another client.
+## Connecting to external libraries and tools
 
-<Image
-  alt="New migration files trigger migrations on the preview instance."
-  src={{
-    dark: '/docs/img/guides/database/connecting-to-postgres/how-connection-pooling-works.png',
-    light:
-      '/docs/img/guides/database/connecting-to-postgres/how-connection-pooling-works--light.png',
-  }}
-  caption="Connecting to the database directly vs using a Connection Pooler"
-  zoomable
-/>
+Beyond the [Supabase client-libraries](https://supabase.com/docs/guides/api/rest/client-libs), Supabase provides three connection strings for direct database access, compatible with all postgres libraries, such as [Prisma](/docs/guides/database/prisma) and [Drizzle](/docs/guides/database/drizzle), as well as tools like [PSQL](/docs/guides/database/psql) and [pgAdmin](/docs/guides/database/pgadmin).
```

Comment:
Which page is better for client libraries https://supabase.com/docs/guides/api/rest/client-libs) or [the one]https://supabase.com/docs#client-libraries (used above)?

Reply:
We actually have a page for this now at https://supabase.com/docs/reference, just hidden behind an old redirect 🤦🏼‍♀️ 

PR to add it back here, then we can use that link instead: https://github.com/supabase/supabase/pull/29264

Reply:
I added it to the request, but I'm curious if the new client page should also feature link to the community libraries